src/ert/shared/models/ensemble_smoother.py

```diff
@@ -44,7 +44,7 @@ async def run(self, evaluator_server_config: EvaluatorServerConfig) -> None:
         )
         await self.dispatch(event)
 
-        self._checkMinimumActiveRealizations(prior_context)
+        self._checkMinimumActiveRealizations(prior_context.mask.count(True))
```

Comment:
Can you not do `len(prior_context.active_realizations)`?

Reply:
No, because that will be equal to num_realizations, and some could be deactivated, the mask is a `List[bool]` so counting the number of `True`